### ISSUE TITLE
Blaze frequently ignore bellows pushing them

### ISSUE THREAD

**Issue by BeeFox-sys** [MehVahdJukaar/Supplementaries#740]
Mod Version: v1.20-2.5.15
Minecraft Version: 1.20

Blaze frequently ignore bellows, this does not apply to all blaze, but i'd say 1 in 10 blaze just do not get pushed by bellows.
This can be replicated by setting up bellows around some blaze spawners to push the blazes away. After about 10 - 15 minutes most blazes will be sitting in the airflow of bellows but being uneffected

![image](https://github.com/MehVahdJukaar/Supplementaries/assets/42090331/493fb8d6-c9e7-4c71-98c0-3d5a6913f6dd)
![image](https://github.com/MehVahdJukaar/Supplementaries/assets/42090331/1219da18-f484-43d8-b74e-6b6dd90c93b4)

**Comment by MehVahdJukaar** [comment]
Could he that they are actively walking away from them

**Comment by BeeFox-sys** [comment]
@MehVahdJukaar no, while sometimes they do these blaze are not moving at all. the ones that are actively walking towards them do get pushed back then move forward and repeat that, these stuck blazes have zero motion at all

**Comment by MehVahdJukaar** [comment]
Must be some quirk of blazes themselves then, something related to their hovering behsvior that makes them not react to pushes

**Comment by Xelbayria** [comment]
Is this issue related to Supplementaries?
